src/main/java/reposense/model/Author.java

```diff
@@ -5,30 +5,103 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * Represents a Git Author.
+ */
 public class Author {
+
     public static final String UNKNOWN_AUTHOR_GIT_ID = "-";
 
-    private String gitId;
+    private static final String MESSAGE_ILLEGAL_GIT_ID = "The provided Git ID, %s, contains illegal characters.";
+    private static final String MESSAGE_ILLEGAL_DISPLAY_NAME =
+            "The provided display name, %s, contains illegal characters.";
+    private static final String MESSAGE_ILLEGAL_AUTHOR_ALIAS =
+            "The provided author alias, %s, contains illegal characters.";
+    private static final String MESSAGE_UNCOMMON_GLOB_PATTERN =
+            "The provided ignore glob, %s, uses uncommon pattern.";
```

Comment:
Cause glob is a pattern rather than a string. Hence I named it pattern.
`uncommon` is because almost anything under the sun can be inserted as glob and it is still valid.
name is `illegal` because we dont accept `uncommon` as input, hence its an IllegalArgument to us.